lib/ActivityStreams.js

```diff
@@ -33,11 +33,11 @@ function ActivityStreams(options = {}) {
   this._setupPageMod();
   this._setupListeners();
   this._setupButton();
-  this._hideAppURLs();
-  NewTabURL.override(this.options.pageURL);
+  NewTabURL.override(`${this.options.pageURL}#/`);
```

Comment:
Actually, since this is in the parent process, I'm probably wrong, and this is probably synchronous.